### ISSUE TITLE
Spinal hypomyelination

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6982]
**Preferred term label:**
Spinal hypomyelination

**Synonyms**

**Definition (free text, please give PubMed ID)**
Reduced amount of myelin in the spinal cord resulting from defective myelinogenesis.

Todo -- distinguish from myelin destruction.

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0003429

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
OMIM:615281 - revised entire entry

**Comment by pnrobinson** [comment]
done
